apstools/plans/xpcs_mesh.py

```diff
@@ -0,0 +1,237 @@
+from bluesky import plan_stubs as bps
+from bluesky import utils
+from bluesky import plan_patterns
+from bluesky import preprocessors as bpp
+
+import inspect
+from itertools import zip_longest
+from collections import defaultdict
+import os
+
+try:
```

Comment:
Neither `cytools` or `toolz` is listed as a project requirement.  Should not assume that other packages will bring them in.  Add one or the other to this section (sorted alphabetically): https://github.com/BCDA-APS/apstools/blob/b0868619ba9450670903e26438af4dabf117c1af/pyproject.toml#L54-L72

Reply:
added cytools to pyproject.toml